src/main/java/net/sf/jabref/logic/net/URLDownload.java

```diff
@@ -119,6 +118,20 @@ private URLConnection openConnection() throws IOException {
             }
 
         }
+
+        // normally, 3xx is redirect
+        int status = connection.getResponseCode();
+        if (status != HttpURLConnection.HTTP_OK) {
+            if (status == HttpURLConnection.HTTP_MOVED_TEMP
+                    || status == HttpURLConnection.HTTP_MOVED_PERM
+                    || status == HttpURLConnection.HTTP_SEE_OTHER) {
+                // get redirect url from "location" header field
+                String newUrl = connection.getHeaderField("Location");
+                // open the new connnection again
+                connection = (HttpURLConnection) new URLDownload(newUrl).openConnection();
```

Comment:
It sounds weird, but it is the truth!
Explaination from the sun java engineers: (Security Reasons)
http://bugs.java.com/bugdatabase/view_bug.do?bug_id=4620571